.travis.yml

```diff
@@ -1,32 +1,35 @@
 language: scala
 scala:
-- 2.12.2
+  - 2.12.2
 stages:
-- name: test
-  if: "(branch = master OR branch = simulation_schemas OR branch = ionchannel) AND
-    type = pull_request"
-- name: gensite
-  if: branch = docs AND type = pull_request
-- name: ghpages
-  if: branch = docs AND type = push
+  - name: test
+    if:
+      "(branch = master OR branch = simulation_schemas OR branch = ionchannel) AND
+      type = pull_request"
+  - name: gensite
+    if: branch = docs AND type = pull_request
+  - name: ghpages
+    if: branch = docs AND type = push
 jobs:
   include:
-  - stage: test
-    script:
-    - sbt -javaOptions=.jvmopts compile
-    - sbt -javaOptions=.jvmopts test
-  - stage: gensite
-    script: sbt clean scalafmtCheck scalafmtSbtCheck paradox
-  - stage: ghpages
-    script: sbt clean makeSite ghpagesPushSite
+    - stage: test
+      script:
+        - sbt -javaOptions=.jvmopts compile
+        - sbt -javaOptions=.jvmopts test
+    - stage: gensite
+      script: sbt clean scalafmtCheck scalafmtSbtCheck paradox
+    - stage: ghpages
+      script:
+        - npm run build --prefix src/main/product-page-src
```

Comment:
Excellent. We just need to fix the encrypted keys errors. I think I erased the encrypted keys (-K $encrypted_b14561f0d260_key -iv $encrypted_b14561f0d260_iv) values in travis when I was working on releasing neuroshapes in bintray.